### PR TITLE
[docs] gentle core walkthrough

### PR DESCRIPTION
**Pull request by maxpumperla**
this is the revert-revert of #32814, which has already been reviewed quite a lot. This PR fixes the syntax error we introduced in the last commit, which broke the build.

### CODE REVIEW

File: doc/source/ray-core/examples/gentle_walkthrough.ipynb [hunk 1]
```diff
@@ -0,0 +1,776 @@
...
+     "output_type": "stream",
+     "text": [
+      "2022-12-20 13:52:34,632\tINFO worker.py:1529 -- Started a local Ray instance. View the dashboard at \u001B[1m\u001B[32m127.0.0.1:8265 \u001B[39m\u001B[22m\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Runtime: 4.54 seconds, data:\n",
```
Comment (jjyao) [inline doc/source/ray-core/examples/gentle_walkthrough.ipynb:286]: Can we fix this now :)

**Comment by richardliaw** [comment]
LMK when @jjyao approves

File: doc/source/ray-core/examples/gentle_walkthrough.ipynb (hunk 1 above)
Comment (maxpumperla) [reply doc/source/ray-core/examples/gentle_walkthrough.ipynb]: haha, thanks.

**Comment by maxpumperla** [comment]
@richardliaw @jjyao please approve + merge.

**Review by richardliaw — APPROVED** [review]

**Pull request merged**